kano_greeter/last_user.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import os
+
+LAST_USER_PATH = '/var/lib/lightdm/.cache/kano-greeter/'
+LAST_USER_FILE = 'last-user'
```

Comment:
Would it make sense to change `LAST_USER_FILE` to be `os.path.join(LAST_USER_PATH, 'last-user')` instead to simplify the file IO calls later?